scripts/InRangeEvents.ts

```diff
@@ -1,51 +1,31 @@
-// Run: npx hardhat run scripts/InRangeEvents.ts --network childTest
-import { ethers } from "hardhat";
+// Run: npx hardhat run scripts/InRangeEvents.ts --network childTest || npm run InRangeEvents:childTest
+import { getEventsByFilters } from "./_helper";
 
-// eslint-disable-next-line no-unused-vars
-const EXAMPLE_BLOCK = 1300500; // 17:23 19.04.2024 1300500 block
-
-// Input parameters for the function:
+// Input parameters for the script:
 const CONTRACT_ADDRESS = "0x0000000000000000000000000000000000000105";
 const CONTRACT_NAME = "RewardPool";
 const EVENT = "ValidatorRewardDistributed(address indexed validator, uint256 amount)";
-const VALIDATOR_ADDRESS = "PUT_ADDRESS_HERE";
-const YOUR_START_BLOCK = EXAMPLE_BLOCK - 1000;
-const YOUR_END_BLOCK = EXAMPLE_BLOCK;
+const VALIDATOR_ADDRESS = process.env.VALIDATOR_ADDRESS_FOR_SCRIPTS;
```

Comment:
There is no check is the variable available

Reply:
Added checks everywhere if the env variable is missing